integrations/terraform/README.md

```diff
@@ -7,22 +7,16 @@ Please, refer to [official documentation](https://goteleport.com/docs/admin-guid
 ## Development
 
 1. Install [`protobuf`](https://grpc.io/docs/protoc-installation/).
-2. Install [`protoc-gen-terraform`](https://github.com/gravitational/protoc-gen-terraform) @v2.2.0.
+2. Install [`protoc-gen-terraform`](https://github.com/gravitational/protoc-gen-terraform).
 
-    ```go install github.com/gravitational/protoc-gen-terraform@08768262d29336b8ae0915ef41bb6d9768518c66```
+    ```go install github.com/gravitational/protoc-gen-terraform@latest```
```

Comment:
Wasn't sure about this, but the failure message I got when I didn't have protoc-gen-terraform installed was that I needed to install `v3.0.0`, even though the README here said v2.2.0.

There doesn't appear to be a v3.0.0 tag, but installing latest worked for me.